scim2-sdk-client/src/main/java/com/unboundid/scim2/client/requests/RequestBuilder.java

```diff
@@ -178,12 +178,23 @@ static ScimException toScimException(final Response response)
         ScimException.createException(response.getStatus(), null) :
         ScimException.createException(errorResponse, null);
       response.close();
+
       return exception;
     }
     catch(ProcessingException ex)
     {
+      String errorMessage = ex.getMessage();
+
+      if (ex.getClass().getSimpleName().equals("MessageBodyProviderNotFoundException"))
```

Comment:
MessageBodyProviderNotFoundException is a Jersey exception, so this is unlikely to work if the client is backed by some other JAX-RS client library, such as RestEasy. We want to avoid any runtime dependencies on specific JAX-RS implementations. In the short term, I think we're better off removing this conditional so that this applies to any kind of ProcessingException. 

The problem is that some SDK users might want to know if a deserialization problem or similar problem has occurred. For example, they could be testing a SCIM 2 server implementation, and they actually do want the server to send back a proper SCIM error response. In the long term, we can consider updating the RequestBuilder so that the SDK user can decide how the client will handle cases like this.

It might also make sense for us to start using something like SLF4J in the SDK, so that SDK users have a way to optionally capture exception details that would otherwise get swallowed.